### PR TITLE
feat: hosting(azure)

### PR DESCRIPTION
**Pull request by bketelsen**
Add configuration files for azure static web apps hosting of content repository

### CODE REVIEW

**Comment by bketelsen** [comment]
fixes #29 
fixes #28

File: cmd/blox.cue [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
```
Comment (rawkode) [inline cmd/blox.cue:1]: Should this be here?

**Review by rawkode — CHANGES_REQUESTED** [review]

File: cmd/blox.cue (hunk 1 above)
Comment (bketelsen) [reply cmd/blox.cue:1]: the file?

Comment (rawkode) [reply cmd/blox.cue:1]: Oh, it's part of init!

**Review by rawkode — APPROVED** [review]

**Pull request merged**